### PR TITLE
Update LLO config prefix to 0x6

### PR DESCRIPTION
**Pull request by austinborn**

### CODE REVIEW

**Review by Fletch153 — APPROVED** [review]

**Comment by austinborn** [comment]
Tests will fail until we get the upgrade in [libocr](https://github.com/smartcontractkit/libocr/pull/63)

File: core/services/relay/evm/mercury/config_digest.go
Comment (austinborn) [inline core/services/relay/evm/mercury/config_digest.go:64]:
```suggestion
	if types.ConfigDigestPrefixMercuryV02 != 6 {
```

File: core/services/relay/evm/mercury/offchain_config_digester.go
Comment (austinborn) [inline core/services/relay/evm/mercury/offchain_config_digester.go:67]:
```suggestion
	return types.ConfigDigestPrefixMercuryV02
```

File: core/services/relay/evm/mercury/config_digest.go
Comment (jmank88) [inline core/services/relay/evm/mercury/config_digest.go:67]: Can this be removed? If not, maybe a comment?

Comment (austinborn) [reply core/services/relay/evm/mercury/config_digest.go:67]: Yeah I can add a comment here

Comment (austinborn) [reply core/services/relay/evm/mercury/config_digest.go:67]: Done!

File: core/services/relay/evm/mercury/offchain_config_digester.go
Comment (austinborn) [inline core/services/relay/evm/mercury/offchain_config_digester.go:67]:
```suggestion
	return 6 // TODO change to types.ConfigDigestPrefixMercury when it is available in libocr
```

**Review by jmank88 — DISMISSED** [review]

**Comment by austinborn** [comment]
Fixing linting now

**Review by samsondav — DISMISSED** [review]

File: contracts/src/v0.8/llo-feeds/Verifier.sol
Comment (kaleofduty) [inline contracts/src/v0.8/llo-feeds/Verifier.sol:333]:
```suggestion
    // 0x006 corresponds to ConfigDigestPrefixMercuryV02 in libocr
    uint256 prefix = 0x0006 << (256 - 16); // 0x000600..00
```

File: core/services/relay/evm/mercury/config_digest.go
Comment (kaleofduty) [inline core/services/relay/evm/mercury/config_digest.go:70]:
```suggestion
	configDigest[1] = 6
```
!!!

File: core/services/relay/evm/mercury/config_digest.go
Comment (kaleofduty) [inline core/services/relay/evm/mercury/config_digest.go:68]: Please uncomment already, unless you're waiting for libocr to be merged before you merge this

**Review by kaleofduty — CHANGES_REQUESTED** [review]

**Comment by jmank88** [comment]
#9350 will unblock this

**Review by jmank88 — APPROVED** [review]

**Review by kaleofduty — APPROVED** [review]
image
approved

**Review by jmank88 — APPROVED** [review]

**Pull request merged**
